### PR TITLE
[MRG] update questionnaires.md

### PR DESCRIPTION
**Pull request by JoseAlanis**
- adds proposal for the evaluation method of the training sessions

### CODE REVIEW

**Comment by JoseAlanis** [comment]
what about the netlify checks? Are they important now that we're on github-pages? Did I miss something?

File: lecture/questionnaires.md [hunk 1]
```diff
@@ -1 +1,104 @@
...
+We believe this to be even more so in the context of scientific training, where
+trainers and trainees can make use of feedback loops and
+"goal-attainment-checks" to identify hindrances and the aids needed to overcome
+them. 
+
+Here, we provide a standardised method for the assessment of this type of
+characteristics as students progress throughout the development of their
+own projects.
+
+We consider it to be crucial to assess already a the beginning
```
Comment (PeerHerholz) [inline lecture/questionnaires.md:16]: _at_

File: lecture/questionnaires.md [hunk 2]
```diff
@@ -1 +1,104 @@
...
+of a training session and/or lecture how much students know about the topic and
+how confident the feel about applying these methods on the own, previous
```
Comment (PeerHerholz) [inline lecture/questionnaires.md:18]: _their_

Comment (PeerHerholz) [reply lecture/questionnaires.md:18]: _prior_

**Review by PeerHerholz — CHANGES_REQUESTED** [review]
I would add a few lines explaining that there will be two general types of questionnaires: a general overarching one that will be conducted at the beginning, mid-term and end of the lecture, as well as topic specific ones each week. The corresponding questions should be added. The same holds true for pointers re the implementation: google forms (provide example), etc. .

File: lecture/questionnaires.md [hunk 3]
```diff
@@ -1 +1,109 @@
...
+
+We consider it to be crucial to assess already at the beginning
+of a training session and/or lecture how much students know about the topic and
+how confident they feel about applying these methods on their own, prior
+to receiving any formal training. This provides a estimate of the students
+baseline level, making it easier to evaluate their individual progress during
+the project.
+
+The assessment consist of:
+1) A general evaluation of the participants satisfaction and progress
```
[the remark this answers is not in the document]
Comment (JoseAlanis) [reply lecture/questionnaires.md:24]: this was WIP after @PeerHerholz's comments. I'll try to make something out of the section that satisfies everybody :)

File: lecture/questionnaires.md [hunk 4]
```diff
@@ -1 +1,177 @@
...
+
+| **no, not ar all** |     |     |     |     |     | **yes, entirely** |
+|:------------------:|-----|-----|-----|-----|-----|:-----------------:|
+| (1)                | (2) | (3) | (4) | (5) | (6) | (7)               |
+
+**2.9)** What was the most challenging part of the training/lecture?
+Did something in particular helped you master these challenges? If you
+didn't experience any difficulties, please tell us what the most 
+helpful/insightful part of the training/lecture was for you (< 100 words):
+
```
Comment (PeerHerholz) [inline lecture/questionnaires.md:157]: should 2.9) include the question if something was missing or should this be an another question? E.g. "Was there content missing from the lecture, aspects you thought would be included or help you understand *<topic_of_interest>*?

Comment (JoseAlanis) [reply lecture/questionnaires.md:164]: Oh yes, sorry, got lost somewhere, I'll add it right away

**Review by PeerHerholz — APPROVED** [review]

File: lecture/questionnaires.md [hunk 5]
```diff
@@ -6,65 +6,122 @@ lab members, co-authors, and, in the end, the scientific community, is of
...
+Optionally, questions 1.2) and 1.3) can be presented a second time (e.g., at
+the end of a section) to provide a comparison.
+
+2) The second part of the assessment provides a more fine grained evaluation of
+the students' degree of progress and difficulties during the sessions. The
+questions 2.1) and 2.2) are designed to be answered by the students
+prior to any specific training session and allow a closer look at the
+their degree of "preparedness" and confidence in the topic.
+Questions 2.3) to 2.9) target students' satisfaction with didactics and
+content, but also experienced difficulties during the training.
```
Comment (vogelbac) [inline lecture/questionnaires.md:38]: instead of training call it session? or training session?

**Pull request merged**